mteb/tasks/Clustering/dan/WikiClusteringP2P.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import annotations
+
+from mteb.abstasks.AbsTaskClustering import AbsTaskClustering
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class WikiClusteringP2P(AbsTaskClustering):
```

Comment:
Would make it a multilingual task, you can use the `MultilingualTask` for that. That way you can combine all the tasks into 1